### ISSUE TITLE
"NameError: name '_model' is not defined" when running dynamic example scenarios

### ISSUE THREAD

**Issue by lasnik** [BerkeleyLearnVerify/Scenic#29]
Hi, thanks for this awesome work! I think it's a step in a very interesting and underexplored direction.

I'm currently familiarizing myself with using scenic in conjunction with carla and running a few of the example scenarios. Unfortunately those with dynamic components don't seem to be working out of the box. See below for specifics.

Env:
* Senic version: master@fbd254c895d295c8139465d483538b437fed4b7
* Carla version: 0.9.11
* Python version: 3.8.5
* OS: Ubuntu 20.04

Traceback:
```python
scenic examples/carla/Carla_Challenge/carlaChallenge4.scenic -b --simulate --model scenic.simulators.carla.model --time 200

Beginning scenario construction...
  Loading cached version of road network...
  Loaded cached network in 0.17 seconds.
pygame 2.0.1 (SDL 2.0.14, Python 3.8.5)
Hello from the pygame community. https://www.pygame.org/contribute.html
  Pruning scenario...
  Pruned scenario in 1.931e-05 seconds.
Scenario constructed in 1.26 seconds.
  Connecting to CARLA...
  Map loaded in simulator.
  Generated scene in 531 iterations, 1.239 seconds.
  Beginning simulation of top-level scenario...

Traceback (most recent call last):
  File "/home/nik/.virtualenvs/carla0911py38/bin/scenic", line 2, in <module>
    from scenic.__main__ import dummy
  File "/home/nik/workspace/Scenic/src/scenic/__main__.py", line 158, in <module>
    success = runSimulation(scene)
  File "/home/nik/workspace/Scenic/src/scenic/__main__.py", line 138, in runSimulation
    result = errors.callBeginningScenicTrace(
  File "/home/nik/workspace/Scenic/src/scenic/core/errors.py", line 170, in callBeginningScenicTrace
    return func()
  File "/home/nik/workspace/Scenic/src/scenic/__main__.py", line 139, in <lambda>
    lambda: simulator.simulate(scene, maxSteps=args.time, verbosity=args.verbosity,
  File "/home/nik/workspace/Scenic/src/scenic/core/simulators.py", line 39, in simulate
    result = simulation.run(maxSteps)
  File "/home/nik/workspace/Scenic/src/scenic/core/simulators.py", line 130, in run
    actions = behavior.step()
  File "/home/nik/workspace/Scenic/src/scenic/core/dynamics.py", line 412, in step
    actions = self._runningIterator.send(None)
  File "/home/nik/workspace/Scenic/examples/carla/Carla_Challenge/carlaChallenge4.scenic", line 29, in makeGenerator
    do CrossingBehavior(ego, speed, threshold)
  File "/home/nik/workspace/Scenic/src/scenic/core/dynamics.py", line 49, in _invokeSubBehavior
    yield from self._invokeInner(agent, subs)
  File "/home/nik/workspace/Scenic/src/scenic/core/dynamics.py", line 433, in _invokeInner
    yield from sub._runningIterator
  File "/home/nik/workspace/Scenic/src/scenic/simulators/carla/behaviors.scenic", line 56, in makeGenerator
    if isinstance(self, _model.Walks):
NameError: name '_model' is not defined
```

Any help would be appreciated!

**Comment by dfremont** [comment]
Thanks for the detailed report! Indeed, a bad merge broke the `CrossingBehavior`, which is used in that scenario. I've pushed a fix (in 49f76e1); please let me know if this scenario (or any other example scenario) still isn't working for you.

(By the way, there's no need for the `--model` command-line option when running the CARLA example scenarios, since they are written to use the CARLA world model by default. You only need that option when running the simulator-agnostic examples in `examples/driving`.)

**Comment by lasnik** [comment]
Thanks for the quick response. Quickly checked on two scenarios, seems to be resolved!
